traits/tests/test_editor_factories.py

```diff
@@ -62,14 +62,6 @@ class TestDateEditor(SimpleEditorTestMixin, unittest.TestCase):
     factory_name = "date_editor"
 
 
-@unittest.skip("DateTimeEditor does not exist in traitsui")
```

Comment:
I think we want to still run `test_editor` (else `datetime_editor` is untested), but not `test_editor_caching`. There are various ways we can arrange for that to happen: e.g., setting `cache_name=None` for `TestDateTimeEditor` and then modifying `SimpleEditorTestMixin` to skip `test_editor_caching` in that case.